helm/aio-connector-multidataset/templates/aio-connector-device.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: namespaces.deviceregistry.microsoft.com/v1beta1
+kind: Device
+metadata:
+  name: {{ .Values.device.name | default "device-001" }}
+  namespace: {{ default "azure-iot-operations" }}
+spec:
+  attributes:
+    deviceId: {{ .Values.device.id | default "device-001" }}
+    deviceType: {{ .Values.device.type | default "device-type-001" }}
```

Comment:
Think we can remove the defaults which are unlikely to work in most cases when used by customers.

Reply:
Defaults removed